app/views/newflow/login/login_form.html.erb

```diff
@@ -93,7 +93,8 @@
                     <%= f.submit I18n.t(:"login_signup_form.continue_button"),
                                 disable_with: I18n.t(:"login_signup_form.continue_button"),
```

Comment:
does this need both the `disable_with` and the newly added `disabled` in the data?

Reply:
I'm actually not sure what disable_with does, but it looks like it provides a message to display when the item is disabled, whereas the disabled setting makes it disabled.